charts/flyte/templates/propeller/configmap.yaml

```diff
@@ -46,5 +46,5 @@ data:
       max_size_mbs: {{ .Values.flytepropeller.cacheSizeMbs }}
       target_gc_percent: 70
 {{- with .Values.configmap.task_logs }}
-  task_logs.yaml: | {{ tpl (toYaml .) $ | nindent 4 }}
```

Comment:
Was the tpl to generate the config?

Reply:
Helm includes a [tpl function](https://v2.helm.sh/docs/developing_charts/#using-the-tpl-function) that expands template content in a string.

```
        - kubernetes-template-uri: http://localhost:30082/#/log///pod?namespace=
        + kubernetes-template-uri: http://localhost:30082/#/log/{{ .namespace }}/{{ .podName
          }}/pod?namespace={{ .namespace }}
```
propeller need URL with substitute variable and it will add those variable in run time.